content/docs/user-guide/building/requirements.md

```diff
@@ -0,0 +1,40 @@
+# Requirements
+
+Get `pip` based and `unix` based requirements for your model.
```

Comment:
I think it would be useful to start with the use case (eventually we need to apply this to all pages like this in #188). E.g.: "When you have a MLEM model and want to use it, you need to install the right packages first..."

The idea is to start with the situation user is familiar with. The first thought of this when he opens the page should be: ok, I know this, this is my case I'm trying to solve. What I need to do for it? Get requirements and install them with pip. Alright! Reading the rest of the page... 📖

Reply:
done